src/components/Tabs/Annotation/AnnotationTrackThumbnail.tsx

```diff
@@ -240,6 +266,10 @@ export default function AnnotationTrackThumbnail(inputProps: AnnotationTrackThum
     }
     const selectedTimes = ids.map((id) => dataset.getTime(id));
     const selectedIntervals = getIntervals(selectedTimes);
+
+    const bgTimes = (props.bgIds ?? []).map((id) => dataset.getTime(id));
```

Comment:
non-blocking question: will getTime ever return `null` or `undefined`? If yes, do we have this handled somewhere?